packages/myst-frontmatter/src/frontmatter/types.ts

```diff
@@ -1,4 +1,5 @@
 import type { CreditRole } from 'credit-roles';
+import { SupportOptionRange } from 'prettier';
```

Comment:
Should this be here?

Reply:
looks like a VSCode overzealous-auto-add! removing.